docs/release-notes/self-hosted/self-hosted-v13.0.0.md

```diff
@@ -0,0 +1,146 @@
+---
+rss_title: Codacy release notes RSS feed
+rss_href: /feed_rss_created.xml
+description: Release notes for Codacy Self-hosted v13.0.0.
+codacy_tools_version_old: https://github.com/codacy/codacy-tools-sh/releases/tag/sh-1.2.6
+codacy_tools_version_new: https://github.com/codacy/codacy-tools-sh/releases/tag/sh-1.2.7
+---
+
+# Self-hosted v13.0.0
+
+These release notes are for [Codacy Self-hosted v13.0.0](https://github.com/codacy/chart/releases/tag/13.0.0){: target="_blank"}, released on October 26, 2023.<!-- TODO Update release date -->
+
+📢 [Visit the Codacy roadmap](https://roadmap.codacy.com) and <span class="skip-vale">let us know</span> your feedback on both new and planned product updates!
+
+## Upgrading Codacy Self-hosted
+
+Follow the steps below to upgrade to Codacy Self-hosted v13.0.0:
+
+1.  Check the [release notes for all Codacy Self-hosted versions](../index.md#self-hosted) **between your current version and the most recent version** for breaking changes and follow the instructions provided <span class="skip-vale">carefully</span>.
+
+1.  Follow the instructions to [upgrade your Codacy Self-hosted instance](https://docs.codacy.com/v13.0/chart/maintenance/upgrade/).
+
+1.  Update your Codacy command-line tools to the following versions:
+
+    -   [Codacy Analysis CLI 7.9.4](https://github.com/codacy/codacy-analysis-cli/releases/tag/7.9.4)
+    -   [Codacy Coverage Reporter 13.13.8](https://github.com/codacy/codacy-coverage-reporter/releases/tag/13.13.8)
+
+## Support for ingress-nginx 4.8.3
```

Comment:
@tfrgoncalves @loliveiracodacy Can you please validate this **Support for ingress-nginx 4.8.3** section? 🙏

Reply:
For correctness sake, 4.8.3 is the chart version which installs the Nginx version (1.9.4) fixing the issues.
Overall we should refer to it as "Support for ingress-nginx chart 4.8.3". I wouldn't propose mentioning the Nginx fix version (1.9.4) for simplicity,

Reply:
```suggestion
## Support for ingress-nginx chart 4.8.3
```